sched/scheduler/jobStatus_test.go

```diff
@@ -0,0 +1,171 @@
+package scheduler
+
+import (
+	"fmt"
+	"github.com/golang/mock/gomock"
+	"github.com/leanovate/gopter"
+	"github.com/leanovate/gopter/prop"
+	s "github.com/scootdev/scoot/saga"
+	"github.com/scootdev/scoot/sched"
+	"testing"
+)
+
+func Test_GetJobStatus_Error(t *testing.T) {
```

Comment:
Agree added a meeting to the calendar on Friday for us to discuss this and come with a path forward.